tests/nested/core/core20-kernel-reseal/task.yaml

```diff
@@ -36,7 +36,9 @@ prepare: |
   remote.push new-pc-kernel.snap
 
 execute: |
-  SEALED_KEY_MTIME_1="$(remote.exec sudo stat --format="%Y" /run/mnt/ubuntu-boot/device/fde/ubuntu-data.sealed-key)"
+  if os.query is-ubuntu-lt 24.04; then
+    SEALED_KEY_MTIME_1="$(remote.exec sudo stat --format="%Y" /run/mnt/ubuntu-boot/device/fde/ubuntu-data.sealed-key)"
```

Comment:
should we use luksDump in some of these tests in the else case? same for other tests were it seems we will do less checks in the luks metadata case